assets/schema/ensemble_schema.json

```diff
@@ -3765,6 +3759,96 @@
         }
       }
     },
+    "BottomNavBarMenuItems-Payload": {
+      "properties": {
+        "styles": {
+          "type": "object",
+          "properties": {
+            "color": {
+              "$ref": "#/$defs/typeColors",
+              "description": "Unselected icon color, starting with '0xFF' for full opacity e.g 0xFFCCCCCC"
+            },
+            "selectedColor": {
+              "$ref": "#/$defs/typeColors",
+              "description": "Selected icon color, starting with '0xFF' for full opacity e.g 0xFFCCCCCC"
+            },
+            "backgroundColor": {
+              "$ref": "#/$defs/typeColors",
+              "description": "Background color, starting with '0xFF' for full opacity e.g 0xFFCCCCCC"
+            },
+            "floatingBackgroundColor": {
+              "$ref": "#/$defs/typeColors",
+              "description": "Floating item background color, starting with '0xFF' for full opacity e.g 0xFFCCCCCC"
+            },
+            "floatingIconColor": {
+              "$ref": "#/$defs/typeColors",
+              "description": "Floating item icon color, starting with '0xFF' for full opacity e.g 0xFFCCCCCC"
+            }   
+          }
+        },
+        "items": {
+          "type": "array",
+          "description": "List of menu items (minimum 2)",
+          "items": {
+            "type": "object",
+            "properties": {
+              "icon": {
```

Comment:
icon / label / page ... should be on the MenuBase. Specific stuff can be on the BottomNavBar